README.md

```diff
@@ -49,6 +49,16 @@ npm run preview-release
 # http://htmlpreview.github.io/?https://github.com/underdogio/underdogio.github.io/blob/{{preview_branch}}/index.html
 ```
 
+#### OSX Troubleshooting
+If you are on OSX and running into issues (e.g. `globstar` isn't defined and `sed` doesn't support `--in-place`), then please upgrade your `bash` and install `gnu-sed`:
+
+- `bash` upgrade available via shellshock test sites
```

Comment:
I purposely put `/usr/local/bin:/usr/local/sbin` before `/bin` in my path... probably shouldn't, but I've run into issues previously and just always want the homebrew version for most things.